### PR TITLE
add bod route for evaluation results page

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Results - add new menu item for Evaluation Results#351](https://github.com/nerubia/kh360-react/issues/351#issuecomment-1863821748)

FE PR: [update route for evaluation results page #377](https://github.com/nerubia/kh360-react/pull/377)

### CODE REVIEW

File: src/middlewares/bod-middleware.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    return res.status(401).json({ message: "Unauthorized" })
+  }
+  const token = authHeader.split(" ")[1]
+  jwt.verify(
+    token,
+    process.env.ACCESS_TOKEN_SECRET as string,
+    async (error: unknown, decoded: unknown) => {
+      if (error != null) return res.status(403).json({ message: "Forbidden" })
+      const decodedToken = decoded as UserToken
+      if (decodedToken.roles === undefined) {
```
Comment (jatnerubia) [inline src/middlewares/bod-middleware.ts:17]: do we need this? 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**